core/options.js

```diff
@@ -113,10 +117,11 @@ Blockly.Options = function(options) {
   this.maxBlocks = options['maxBlocks'] || Infinity;
   this.pathToMedia = pathToMedia;
   this.hasCategories = hasCategories;
+  this.hasCss = hasCss;
+  this.hasModal = hasModal;
   this.hasScrollbars = hasScrollbars;
-  this.hasTrashcan = hasTrashcan;
   this.hasSounds = hasSounds;
-  this.hasCss = hasCss;
+  this.hasTrashcan = hasTrashcan;
```

Comment:
Looks like you could revert back the trash can changes and maintain the `hasCss` flag.